src/components/FormWizard.vue

```diff
@@ -45,7 +45,7 @@
       </div>
     </div>
 
-    <div class="wizard-card-footer">
+    <div class="wizard-card-footer" v-if="!hideButtons">
```

Comment:
Let's reverse this to `v-if="hideButtons"
This way we keep it backwards compatible and the default behavior will display buttons
`:hide-buttons="true"` will actually hide them
Also it's more natural to say that `true` for `hide-buttons` actually hides buttons than the other way around